app/views/wiki/_header.html.erb

```diff
@@ -31,7 +31,7 @@
 
     <div style="padding-top:8px;text-align:center">
       <span class="d-none d-lg-inline">
-        <span rel="tooltip" title="The number of authors for this page." data-placement="top"><i style="color:#888;" class="fa fa-user"></i> <%= number_with_delimiter(@node.authors.size) %></span>  |
+      <span rel="tooltip" title="The number of authors for this page." data-placement="top"><i style="color:#888;" class="fa fa-user"></i> <%= number_with_delimiter(@node.authors.size) %></span> |
```

Comment:
Good job, we are very close, but there is still a couple of things, like indent this line of code. Git detects every small change.

Reply:
Hey, thanks Infact I was thinking that this indentation might create a problem but wasn't sure. But thanks I will change it now.